### PR TITLE
Suppress exception thrown when deleting scalable target and scaling policy when using DynamoDB local.

### PR DESCRIPTION
**Pull request by Torch3333**
Currently, when deleting a table using the Schema loader on a Dynamo DB local environment, log messages are shown mentioning the scaling policy and scalable target were not found.
```
$ java -jar ~/java/scalardb-schema-loader-3.6.0.jar --config ./database.properties --no-backup --no-scaling --coordinator -D
[main] INFO com.scalar.db.schemaloader.command.SchemaLoaderCommand - Config path: ./database.properties
[main] INFO com.scalar.db.schemaloader.command.SchemaLoaderCommand - Schema path: null
[main] WARN com.scalar.db.storage.dynamo.DynamoAdmin - The scaling policy table/coordinator.state-read is not found.
[main] WARN com.scalar.db.storage.dynamo.DynamoAdmin - The scaling policy table/coordinator.state-write is not found.
[main] WARN com.scalar.db.storage.dynamo.DynamoAdmin - The scalable target table/coordinator.state is not found
[main] WARN com.scalar.db.storage.dynamo.DynamoAdmin - The scalable target table/coordinator.state is not found
[main] INFO com.scalar.db.schemaloader.SchemaOperator - Deleting the coordinator tables succeeded.
```
The same log are also printed when a secondary index is deleted. 

This PR updates the DynamoAdmin to suppress the exception raised by deleting such policies since they cannot be created using a Dynamo DB local environment. 
Consequently, the log warning messages above won't be displayed anymore. 
```
$ java -jar ~/java/scalardb-schema-loader-4.0.0-SNAPSHOT.jar --config ./database.properties --no-backup --no-scaling --coordinator -D
[main] INFO com.scalar.db.schemaloader.command.SchemaLoaderCommand - Config path: ./database.properties
[main] INFO com.scalar.db.schemaloader.command.SchemaLoaderCommand - Schema path: null
[main] INFO com.scalar.db.schemaloader.SchemaOperator - Deleting the coordinator tables succeeded.
```

### CODE REVIEW

**Review by kota2and3kan — APPROVED** [review]
LGTM! Thank you!
It worked fine in my local environment!
```console
$ java -jar ./scalardb-schema-loader-4.0.0-SNAPSHOT.jar --config ./dynamodb.database.properties -f ./sample-schema.json --no-backup --no-scaling --coordinator -D
[main] INFO com.scalar.db.schemaloader.command.SchemaLoaderCommand - Config path: ./dynamodb.database.properties
[main] INFO com.scalar.db.schemaloader.command.SchemaLoaderCommand - Schema path: ./sample-schema.json
[main] INFO com.scalar.db.schemaloader.SchemaOperator - Deleting the table t0 in the namespace s0 succeeded.
[main] INFO com.scalar.db.schemaloader.SchemaOperator - Deleting the coordinator tables succeeded.
```

File: core/src/main/java/com/scalar/db/storage/dynamo/DynamoAdmin.java [hunk 1]
```diff
@@ -995,10 +992,14 @@ private void deleteScalingPolicy(
     for (DeleteScalingPolicyRequest deleteScalingPolicyRequest : deleteScalingPolicyRequestList) {
       try {
         applicationAutoScalingClient.deleteScalingPolicy(deleteScalingPolicyRequest);
-      } catch (Exception e) {
-        if (!(e instanceof ObjectNotFoundException)) {
-          logger.warn(
-              "The scaling policy " + deleteScalingPolicyRequest.policyName() + " is not found.");
+        // Suppress exceptions when the scaling policy does not exist
+      } catch (ObjectNotFoundException e) {
```
Comment (feeblefakie) [inline core/src/main/java/com/scalar/db/storage/dynamo/DynamoAdmin.java:996]:
```suggestion
      } catch (ObjectNotFoundException ignored) {
```

File: core/src/main/java/com/scalar/db/storage/dynamo/DynamoAdmin.java [hunk 2]
```diff
@@ -1010,12 +1011,14 @@ private void deregisterScalableTarget(
...
       try {
         applicationAutoScalingClient.deregisterScalableTarget(deregisterScalableTargetRequest);
-      } catch (Exception e) {
-        if (!(e instanceof ObjectNotFoundException)) {
-          logger.warn(
-              "The scalable target "
-                  + deregisterScalableTargetRequest.resourceId()
-                  + " is not found");
+        // Suppress exceptions when the scalable target does not exist
+      } catch (ObjectNotFoundException e) {
```
Comment (feeblefakie) [inline core/src/main/java/com/scalar/db/storage/dynamo/DynamoAdmin.java:1015]:
```suggestion
      } catch (ObjectNotFoundException ignored) {
```

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!
Left minor suggestions, but it doesn't block it from being merged.

**Review by brfrn169 — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**